### PR TITLE
Initial popUP Ui completed

### PR DESCRIPTION
**Pull request by ajayVibinex**

### CODE REVIEW

File: popup.html [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        <div class="second_box">
+            <img src="./lightIcon.png" alt="" class="light-icon">
+            <p>Vibinex is active, but key features are missing.</p>
+        </div>
+        <div class="mainContent">
+            <p class="secondHeading">Signup now to unlock the following: </p>
+            <ul class="unorderList">
+                <li><span class="list_item">Highlight important PR to reivew</span></li>
+                <li><span class="list_item">Highlight imprortant section of code</span></li>
+                <li><span class="list_item">Get PR review coverage</span></li>
```
Comment (avikalpg) [inline popup.html:27]:
```suggestion
                <li><span class="list_item">Highlight important PRs to reivew</span></li>
                <li><span class="list_item">Highlight important sections of code within PR</span></li>
                <li><span class="list_item">Get PR review coverage</span></li>
```

File: popup.html [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+            </ul>
+        </div>
+        <div class="signUpButton">
+            <a class="signInButton" href="https://www.vibinex.com" target="_blank">Sign in</a>
```
Comment (avikalpg) [inline popup.html:31]: send them directly to the login page.

File: popup.html [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+            <p class="signup_option">Don't have and account? <span class="signUp">
+                    <a href="https://www.vibinex.com" target="_blank">Sign up</a></span></p>
```
Comment (avikalpg) [inline popup.html:33]: Not needed

File: popup_css.css [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+ }
+
+ .list_item {
+     color: black;
+     font-weight: 100;
+     margin-left: 5px;
+
+ }
+
+
```
Comment (avikalpg) [inline popup_css.css:80]: Please maintain consistent spacing between blocks

File: popup.html (hunk 2 above)
Comment (ajayCreations) [reply popup.html:31]: login page url is showing error

**Pull request merged**